README.md

```diff
@@ -13,8 +13,7 @@ This repository contains scripts quickly build out a new tenant and configure it
 
 ### Tooling
 
-> [!NOTE]
-> The following tooling is pre-installed in the [VS Code Dev Container](https://code.visualstudio.com/docs/devcontainers/containers) and it is highly recommended that you use the Dev Container to run the scripts and terraform modules in this repository:
+**_NOTE:_** The following tooling is pre-installed in the [VS Code Dev Container](https://code.visualstudio.com/docs/devcontainers/containers) and it is highly recommended that you use the Dev Container to run the scripts and terraform modules in this repository:
```

Comment:
if we don't generate docs from those document and just keep them in github page, then follow this markdowns:
https://github.com/orgs/community/discussions/16925

Reply:
Thank you for the comment, I try it, but doesn't work in the preview, look:

![image](https://github.com/microsoft/power-platform-terraform-quickstarts/assets/15805460/5a638e7e-7437-43b3-b553-4f3b0c947f34)

Reply:
Thank you for the comment, I try it, but doesn't work in the preview, take a look:

![image](https://github.com/microsoft/power-platform-terraform-quickstarts/assets/15805460/5a638e7e-7437-43b3-b553-4f3b0c947f34)